contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -39,7 +39,7 @@ contract NativeTokenDestination is ITeleporterReceiver, INativeTokenDestination,
     // they can be tracked separately.
     address public constant BURN_FOR_TRANSFER_ADDRESS = 0x0100000000000000000000000000000000000001;
 
-    INativeMinter private immutable _nativeMinter =
+    INativeMinter public constant NATIVE_MINTER =
```

Comment:
Definitely agree with the change to constant vs immutable here. Is there a gas-usage benefit the making it public?

Reply:
no gas benefits, I'm going to change it back to private.

Reply:
nvm updated back to public, to keep in convention with our other Warp declarations